databroker/eventsource/archiver.py

```diff
@@ -37,18 +39,58 @@ def __init__(self, url, timezone, pvs):
         self._descriptors = {}
 
     def insert(self, name, doc):
-        raise NotImplementedError()
+         """
+         Not supported, data archiving is managed via the EPICS Archiver Appliance Toolkit
+         """
+         raise NotImplementedError()
 
     def stream_names_given_header(self, header):
         # We actually don't use the header in this case.
+        """
+        Return a list of user-defined PV names prefixed with 'archiver_'
+
+        Parameters
+        ----------
+        header: Header
+            not used. stream_names are generated from pvs
+        Returns
+        -------
+        list: e.g., [archiver_<user-defined PV name>, ...]
+        """
+            
         return ['archiver_{}'.format(name) for name in self.pvs]
```

Comment:
Can we not tack archiver on the front?  Given that they provided us with nice names on configuration, might as well just use them.